### ISSUE TITLE
rustup docs now optional

### ISSUE THREAD

**Issue by jfols** [stevedonovan/gentle-intro#28]
[Documentation is now opt in](https://users.rust-lang.org/t/psa-rust-documentation-is-now-optional-and-opt-in-when-installing-via-rustup/8072), add a note in [Learning Where to Find the Ropes](http://stevedonovan.github.io/rust-gentle-intro/1-basics.html#learning-where-to-find-the-ropes):

`rustup component add rust-docs`

**Comment by stevedonovan** [comment]
Sure, that's why I added the explicit add component command. Is there something here I'm missing?

**Comment by jfols** [comment]
Ah I see, I skipped over that bit in the intro bc I already installed rust, didn't notice the docs component line, my fault. It threw me off for a second when I got to [Learning Where to Find the Ropes](http://stevedonovan.github.io/rust-gentle-intro/1-basics.html#learning-where-to-find-the-ropes) and nothing happened when I ran the doc command. A reminder might be helpful.

**Comment by stevedonovan** [comment]
Good point!  Personally, I would have voted for opt-out, but we have to work with things as they are.
